orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -703,7 +705,7 @@ void TrainingRunner::RunWithUpdate(VectorString& feed_names,
   // We must join here because main thread needs to access thread-produced
   // fetches and those fetches must be ready.
   pipeline_worker_pool_.JoinAll();
-  for(auto& status : pipeline_worker_pool_.worker_states){
+  for (auto& status : pipeline_worker_pool_.worker_states) {
```

Comment:
Maybe just I didn't find it, but do you need change in training_runner or training_session to use the priority based execution order?

Reply:
see training_runner.cc

static SessionOptions SESSION_OPTION = {
    ExecutionMode::ORT_SEQUENTIAL,     //execution_mode
    **ExecutionOrder::PRIORITY_BASED,    //execution_order**